examples/pipelines_braindecode/braindecode_ShallowFBCSPNet.py

```diff
@@ -18,13 +19,13 @@
 BATCH_SIZE = 64
 SEED = 42
 VERBOSE = 1
-EPOCH = 10
+EPOCH = 5
 PATIENCE = 3
 
 # Create the dataset
-create_dataset = BraindecodeDatasetLoader(drop_last_window=False)
+create_dataset = BraindecodeDatasetLoader()
 
-# Set Shallow Filter Bank CSP Net model
+# Set random Model
 model = ShallowFBCSPNet(
     in_chans=1, n_classes=2, input_window_samples=100, final_conv_length="auto"
```

Comment:
Same remark here, why use `input_window_samples`=100? No real clues in the [paper](https://onlinelibrary.wiley.com/doi/full/10.1002/hbm.23730) or in the [supplementary material](https://onlinelibrary.wiley.com/action/downloadSupplement?doi=10.1002%2Fhbm.23730&file=hbm23730-sup-0001-suppinfo.pdf) (section A2 for FBCSPNet)